### DISCUSSION TITLE
Bold in header

### DISCUSSION THREAD

**Discussion by MontseTelefonica** [Telefonica/mistica-design#571] [Proposals]
Hello! We have found an issue regarding the Moves Start tab header being too bold. The text hierarchy is not so clear now and we think the could try 2 different solutions:
- Option 1: Changing the header body to regular  (I think this one works better)
- Option 2: Choosing a smaller text preset for the header body

<img width="742" alt="image" src="https://user-images.githubusercontent.com/95479574/177279626-8ccd4d6c-b9c6-43ab-b3f2-cbee3cf4ed24.png">

**Comment by yceballost** [comment]
Both options have implications

## Option 1
At this moment, text-preset-5 has not light and regular weights

Add these new weights are quite easy

## Option 2
Change `text-preset-6` →  `text-preset-5` is easier than option 1, BUT, there are other components with text-preset-6 as a title. For example, Feedback Screens, Empty states. 

---

In both options have the same "problem". Do we have to change the feedback and empty state title to `text-preset-5` to keep coherence? Do we have to change the weight on these components to have coherence with the header??

**Reply by yceballost** [reply]
Ibamos a ir por la opción 1 pero eso desmontaría la lógica que se ha montado para movistar evolution.

Esto habrá que resolverlo creando los textos como constantes, es algo complejo y que puede provocar grandes breaking changes y habrá que encontrar el momento para hacerlo.

Como solución para startab, creemos que ese header es custom y no tira de Mística, si fuese así, se podría hacer que el titulo no usara text-preset y de esa manera cambiarle el estilo.

**Comment by yceballost** [comment]
Con la evolución que se plantea del header se podría resolver este problema.
![image](https://user-images.githubusercontent.com/6722153/208654903-053d603a-d9f0-49f4-b740-cce74be11a67.png)

Simplemente esa composición iria dentro del slot y no usaría los elementos que te provee el header (principalmente por el carousel)

Igualmente, haremos pruebas para tener en cuenta este caso en la evolución

Archivo esta discussion
